source/administration/config-settings.rst

```diff
@@ -1183,20 +1183,150 @@ Advanced Logging
 
-- Multiple local file targets: Supports rotation and compression triggered by size and/or duration.
-- Multiple syslogs: Supports local and remote syslog servers, with or without TLS transport.
-- Multiple TCP sockets: TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+System Admins can define multiple log targets to:
 
-These three targets have been chosen to support the vast majority of log aggregators and other log analysis tools without having to install additional software. 
+- Mirror log output to files and log aggregators for redundancy.
+- Log certain entries to specific destinations. For example, all errors could be routed to a specific destination for review.
 
-Beyond the standard log levels (trace, debug, info, panic), discrete log levels can also be specified.
+Additional configuration options include:
 
-.. note::
-   Available discrete log levels are listed in ``mattermost-server:mlog/levels.go``. Logs are recorded asynchronously to reduce latency to the caller. Advanced logging supports hot-reloading of logger configuration.
+- Multiple local file targets: Supports rotation and compression triggered by size and/or duration.
+- Multiple syslogs: Supports local and remote syslog servers, with or without TLS transport.
+- Multiple TCP sockets: TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
 
-This feature's ``config.json`` setting is ``LogSettings.AdvancedLoggingConfig`` which can contain a filespec to another config file, a database DSN, or JSON. Options are outlined in this txt file: `Log Settings Options <https://github.com/mattermost/docs/files/5066579/Log.Settings.Options.txt>`_. Sample config: `Advanced Logging Options Sample.json.zip <https://github.com/mattermost/docs/files/5066597/Advanced.Logging.Options.Sample.json.zip>`_.
+All access to the REST API or CLI is audited. When using Advanced Logging for audit and specifying a file or JSON configuration, System Admins can capture the following auditing in the target configuration:
+
+.. code-block:: none
+
+   "Levels": [
+      {"ID": 100, "Name": "audit-api"},
+      {"ID": 101, "Name": "audit-content"},
+      {"ID": 102, "Name": "audit-permissions"},
+      {"ID": 103, "Name": "audit-cli"},
+   ],
+
+Where:
+
+- ``audit-api`` - Enables output of REST API calls.
+- ``audit-content`` - Enables output of API calls that generate content (e.g. create post, create reaction).
+- ``audit-permissions`` - Enables output of all permissions failures.
+- ``audit-cli`` - Enables output of legacy CLI calls.
+
+Beyond the standard log levels (trace, debug, info, panic), discrete log levels can also be specified. New log levels must be registered in ``mattermost/mattermost-server/shared/mlog/levels.go``.
```

Comment:
Adding new log levels is a developer issue and IMO shouldn't appear in end-user documentation.

Reply:
Good point, @wiggin77. Can a customer with developer knowledge register new log levels within ``mattermost/mattermost-server/shared/mlog/levels.go``? Or is this something that Mattermost would complete on a customer's behalf?

Reply:
I agree to remove this.  Customers can fork or add PRs if they require new log levels.